src/docfx/ErrorCodes.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build
+{
+    internal static class ErrorCodes
+    {
+        public const string ConfigNotFound = "config-not-found";
```

Comment:
>config-not-found [](start = 46, length = 16)

use camel case as in v2? to make it easy to integrate

[https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.Common/Loggers/WarningCodes.cs](https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.Common/Loggers/WarningCodes.cs)

Reply:
v2 is Pascal. The idea is that we can have a webpage like: [https://docfx.github.io/errors-and-warnings#config-not-exist](https://docfx.github.io/errors-and-warnings#config-not-exist), having a *slug* like format is more url friendly. but open for any options.

---
In reply to: [187515915](https://github.com/dotnet/docfx/pull/2758#discussion_r187515915) [](ancestors = 187515915)

Reply:
I am more towards [eslint](https://eslint.org/docs/user-guide/configuring) style config. config itself is camel case, the **rules** are *slugs*